tests/templates/pr-create.md.ejs.js

```diff
@@ -127,28 +127,30 @@ describe("pr-create.md.ejs", function() {
         assert.ok(result.indexOf("require an issue") === -1);
     });
 
-    it("should not mention missing issue or length check when there's one fix commit", function() {
-        var result = ejs.render(TEMPLATE_TEXT, {
-            payload: {
-                sender: {
-                    login: "nzakas"
+    ["Breaking", "Build", "Chore", "Docs", "Fix", "New", "Update", "Upgrade"].forEach(function(type) {
+        it("should not mention missing issue or length check when there's one " + type + " commit", function() {
```

Comment:
I typically prefer to avoid variables in test descriptions, as it makes it just _that_ much harder to find a failing test when all you really have to go off of is that description.  But it's not a big deal here because there's still plenty of plain text to grep for.